### PR TITLE
Fix/dateformat

### PR DESCRIPTION
**Pull request by DeepanshuSarswat**
Fixed date format

### CODE REVIEW

File: src/helperFunctions/getDateInString.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const getDateInString = (date: Date) => {
     const year = date.getFullYear()
-    const month = date.getMonth() + 1;
-    const day = date.getDate();
+    const month = ("0" + (date.getMonth() + 1)).slice(-2);
```
Comment (harshith-venkatesh) [inline src/helperFunctions/getDateInString.ts:3]: Would ternary operator be better suitable here?
Checking the length and adding 0 or not accordingly?
A new dev find it difficult to understand without comments or context

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: src/helperFunctions/getDateInString.ts (hunk 1 above)
Comment (DeepanshuSarswat) [reply src/helperFunctions/getDateInString.ts:3]: Like this?
const a = new Date();
const month = a.getMonth > 9 ? a.getMonth : ("0" + (a.getMonth() + 1)).slice(-2);

Comment (ankushdharkar) [reply src/helperFunctions/getDateInString.ts:3]: @DeepanshuSarswat Please try to use code blocks in the comments. It helps with readability.

@harshith-venkatesh I actually really like the solution used here. A comment would help. I feel a ternary would be an overkill.

```js
const month = ("0" + (date.getMonth() + 1)).slice(-2);
```

Comment (harshith-venkatesh) [reply src/helperFunctions/getDateInString.ts:3]: True, after taking a look at the comment back, I felt the same as well.
However the comment do play an important role here

Comment (ankushdharkar) [reply src/helperFunctions/getDateInString.ts:3]: @harshith-venkatesh I feel at the end of the day, we are not writing code for new devs. We are writing code that is easy to read, proven correct (Most important) and reason about.

A new dev should take the time to look up what the language functions do and if they can guarantee that the code will always produce the correct results.

If we always cater to the new dev, we will be stuck writing verbose basic code.

**Review by ankushdharkar — APPROVED** [review]
Very smart and elegant fix! Thanks @DeepanshuSarswat

File: src/helperFunctions/getDateInString.ts (hunk 1 above)
Comment (harshith-venkatesh) [reply src/helperFunctions/getDateInString.ts:3]: Can't disagree with your comment. That intention here was, I know the reason behind this fix.
But a person new to review or with no context should also be able to easily review and approve right.
I can understand, this statement could be easily deciphered, was just presenting my thoughts here🙂👍

**Comment by harshith-venkatesh** [comment]
Good work, I was about to approve this PR as well, it got merged already 😅

**Pull request merged**